doc/manual.rst

```diff
@@ -0,0 +1,116 @@
+
+.. OpenShot Video Editor is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+.. You should have received a copy of the GNU General Public License
+ along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+
+.. Edited September 27, 2020 - [USA]TechDude
+
+User Manual
+===================
+Thank you for your interest in OpenShot Video Editor, the easy to use, quick to 
+learn, and powerful video editor.  OpenShot helps you start any video project
+by importing and editing your video, audio or pictures, adding special effects
+and transitions and sharing your final product on social media sites, personal
+devices or permanent storage.
+
+.. image:: images/title-ops.png
+
+System Requirements
+-------------------
+Video editing benefits from large amounts of memory, modern multi-core CPUs, 
+and fast internal storage.  When editing video, you want the best computer 
+system that you can afford.  Here are the minimum and the recommended system 
+requirements:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Minimum System Requirements
+     - Recommended System Requirements
+   * - System Memory
+        | 4 GB RAM
+     - System Memory 
+        | 32 GB DDR4 RAM
+   * - Multi-core 64-bit processor
+        | AMD x86-64 (2003 - present)
+        | Intel Core i3 (2006 - present)
+     - Multi-core 64-bit processor
+        | AMD x86-64 (2018 - present)
+        | Intel Core i9 (2017 - present)
+   * - Internal Storage Space
+        | 500 MB (for installation)
+        | SATA I, II, or III
+     - Internal Storage Space
+        | 10 GB (for processing)
+        | M.2 PCIe or PCIe NVMe
+   * - Graphics Processing Unit 
+        | Nvidia, AMD, Asus, 
+        | Intel, S3 Graphics, Matrox
+     - Graphics Processing Unit
+        | AMD Radeon or Radeon Pro
+        | NVIDIA TITAN, GeForce or Quadro
+   * - 64-bit Operating System 
+        | Linux Kernel 4.15+
+        | macOS 10.9+ 
+        | Microsoft Windows 7/8/8.1/10 
+     - 64-bit Operating System
+        | Linux Kernel 5.7+
+        | macOS 10.15+
+        | Microsoft Windows 10+
+   * - Display Screen
+        | 14" FHD (1280x720) Laptop Display
+        | 17" FHD (1280x720) Desktop Display
+     - Display Screen
+        | 17" UHD (3840x2160) Laptop Display
+        | 27" UHD (3840x2160) x2 Desktop Displays
+        
+Download and Install
+--------------------
+If you do not yet have OpenShot Video Editor, you can download it from the 
+official |Link|_ for most 64-bit operating systems.  OpenShot is available for
+
+ -  |win| `Microsoft® Windows® <install.html#win-windows>`_
+ -  |lin| `Linux® (most distributions) <install.html#lin-linux>`_
+ -  |mac| `Apple® macOS® <install.html#mac-macos>`_
+
+The latest stable release is version 2.5.1.  With faster performance, huge 
```

Comment:
I would be careful mentioning the version number here... as it will immediately be outdated. Not sure how to version this properly... but I wouldn't go dropping the version around in the text of this guide I don't think.